### ISSUE TITLE
The exported file is garbled

### ISSUE THREAD

**Issue by tianhao** [TablePlus/TablePlus#534]
1. Which driver are you using and version of it (Ex: PostgreSQL 10.0):
    postgresql 9.5
2. Which TablePlus build number are you using (the number on the welcome screen, Ex: build 81):
    110
3. The steps to reproduce this issue:
"Export Table" to csv file

There is a column in the table that is Chinese characters, and the exported CSV file is Chinese garbled.

**Comment by huyphams** [comment]
Hey @tianhao it would be great if you can send me the table structure (click on definition button) to my email: huy@tableplusapp.com

![abc](https://user-images.githubusercontent.com/3654622/42553381-e26c3642-850a-11e8-951e-6daa2b0ebf23.png)

I tested but can't reproduce the bug.
Thanks.

**Comment by tianhao** [comment]
Sorry, there are some errors in the description. The file is not garbled，but it is garbled in Excel

create table and insert:
```
create table test_export( f varchar(128));
insert into test_export(f) values('汉字');
```
Export to file and The content of the file is:
```
"f"
"汉字"
```
File is not garbled，But it is garbled in Excel:
<img width="154" alt="image" src="https://user-images.githubusercontent.com/365540/42554115-32140408-8516-11e8-944f-e9efe1e030df.png">

You can find the solution by google "excel csv bom"

**Comment by huyphams** [comment]
Looks like the bug only happen in Microsoft Excel, I tested on Apple Number, and got the same result:
<img width="733" alt="screen shot 2018-07-11 at 1 37 20 pm" src="https://user-images.githubusercontent.com/3654622/42554565-8ed84b9c-850f-11e8-8ac5-e3132870b5eb.png">

**Comment by tianhao** [comment]
Yes, I thought it was a problem with TablePlus at first, because the files exported by DataGrip are not garbled. But you can optimize it.
